thirdparty_src_checksums.txt

```diff
@@ -23,6 +23,7 @@ d8331bd0f7367c8afd5fcb5f5e85e96868a00fd24b7276fa5fcee1e5575c2662  gflags-2.1.2.t
 f28359aeba12f30d73d9e4711ef356dc842886968112162bc73002645139c39c  glog-0.4.0.tar.gz
 80439c8a92af974e6788347c59d71d59fdf51f808d80d52fabe744eacf4f9cba  glog-0.4.0-yb1.tar.gz
 fd167e7c5bd33fa6d15dcd7e7ddcf08cdf1428d4123d60041951cb4210d97990  glog-0.4.0-yb-2.tar.gz
```

Comment:
Is this line still needed after we move to glog-0.4.0-yb-3.tar.gz?

Reply:
We always seem to keep older version checksums around. @mbautin it safe to remove them?